example/k3d/scripts/smoke-test.bash

```diff
@@ -82,20 +83,16 @@ run() {
   if [[ ! -z "$IMPORT_IMAGES" ]]; then
     echo "--- Importing local images"
 
+    docker pull us.gcr.io/kubernetes-dev/grafana/agent-smoke:latest
```

Comment:
IMO you should be building this locally or pulling it outside of the smoke script instead of having it pull from gcr every time.

Reply:
I agree -- I'll remove this so the end user can decide if they want to build it locally, pull from the registry, etc.